teach/urls.py

```diff
@@ -27,7 +27,8 @@
     url(r'', include('django_browserid.urls')),
     url(r'^admin/', include(teach_admin.urls)),
 
-    url(r'^credly/', include('credly.urls'))
+    url(r'^credly/', include('credly.urls')),
+    url(r'^clubsguides/', include('clubs_guides.urls')),
```

Comment:
I added credly outside of the `/api/` base route because it was effectively a proxy to Credly's API instead of our own, adding `clubsguides` on the root rather than on `/api/clubsguides` may need reconsidering, since there are CORS issues we can solve for "all routes on /api/" that are negated by adding new APIs at the top level.